src/main/java/org/quiltmc/loader/impl/plugin/QuiltPluginManagerImpl.java

```diff
@@ -1585,6 +1585,12 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
 							return FileVisitResult.SKIP_SUBTREE;
 						}
 
+						if (name.startsWith(".")) {  // ignore dot-folders
+							System.out.println(name);
```

Comment:
Debug line spotted!

Reply:
I don't know what you are talking about :trollface: